src/composables/inputListener.ts

```diff
@@ -0,0 +1,30 @@
+// codes we care about
+export enum KeyCode {
+  SHIFT_LEFT = 'ShiftLeft',
+  CTRL_LEFT = 'ControlLeft',
+}
+
+export function useInputListener() {
```

Comment:
isn't this (roughly) the same thing than vue-use's `useKeyModifier`? (https://vueuse.org/core/useKeyModifier/#usekeymodifier)

Reply:
well it's additional code that needs maintenance and might have bugs (don't see any, but ...).

i'm not sure on "other" keys though - as that'd be "onKeyStroke" on vueuse ...